utils/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "utils"
+version = "0.1.0"
+authors = ["jonran"]
```

Comment:
Why the different format for the `authors` value here versus the other Cargo.toml files?